Makefile

```diff
@@ -8,6 +8,7 @@ include $(TOP)/build.mk
 
 SUB_DIRS = utils ercc ecc_enclave ecc tlcc_enclave tlcc examples integration # docs
 PLUGINS = ercc ecc_enclave ecc tlcc_enclave tlcc
+MIN = utils/fabric ercc ecc_enclave ecc
```

Comment:
MIN is a bit MIN as describing purpose ;-)   If i see that correctly, then this is essentially the target to build the "FPC chaincode SDK"? If so, maybe a corresponding name might be better in describing the purpose of the target?

Reply:
ercc should not be part of the FPC SDK - right?! 

I really like the description of FPC SDK and maybe FPC runtime?!